docs/standard/serialization/system-text-json-overview.md

```diff
@@ -11,7 +11,7 @@ helpviewer_keywords:
 ms.assetid: 4d1111c0-9447-4231-a997-96a2b74b3453
 ---
 
-# JSON serialization in .NET - overview
+# JSON serialization and deserialization (marshalling and unmarshalling) in .NET - overview
```

Comment:
> .NET Framework 4.6.1 and later versions

This is unrelated, but maybe we should specify 4.7.2 instead of 4.6.1

From footnote 2 (in https://docs.microsoft.com/en-us/dotnet/standard/net-standard#net-implementation-support):
> For .NET Framework projects that need to use such libraries, we recommend that you upgrade the project to target .NET Framework 4.7.2 or higher.